### DISCUSSION TITLE
When will the labels for the test set released?

### DISCUSSION THREAD

**Discussion by zubatyuk** [snap-stanford/ogb#231] [PCQM4M-LSC]
Hi.
When do you plan to release the labels for the test set? Our team got the validation errors significantly lower than one of the winners (https://github.com/zouxiaochuan/molnet_kddcup2021/blob/master/molnet_arxiv.pdf). We want to analyze what went wrong with the test set prediction.

**Comment by weihua916** [comment]
Great, that's exciting! We will keep the test labels hidden and release the evaluation server for you to submit your model prediction. We will keep the community updated.

**Reply by kirayummy** [reply]
That is helpful for the graph community! And when will you reopen the evaluation server? Thanks!

**Reply by weihua916** [reply]
That’s still to be decided. Hopefully in the next month or so

**Reply by Fairly** [reply]
Hi Weihua, are you still working on the evaluation server? We are so eager to check our new model!

**Reply by weihua916** [reply]
Yes we are making slow progress! Please give us a bit more time, but we will definitely release the eval server sometime soon.

**Comment by cenyk1230** [comment]
Hi @zubatyuk,

Would you please tell us the validation errors that your team got?

**Reply by zubatyuk** [reply]
We had about 0.1 with no signs of serious over fitting.

**Comment by weihua916** [comment]
As mentioned [here](https://groups.google.com/g/open-graph-benchmark/c/gWv-Y5Vg2Cs), the PCQM4M will be deprecated. The new dataset, called PCQM4Mv2, will be introduced by the end of September. It's taking a lot of effort to update/benchmark/debug the datasets after the KDD Cup. Thank you everyone for the patience!
